visual-dotnet/SauceLabs.Visual/GraphQL/CreateSnapshotFromWebDriverIn.cs

```diff
@@ -26,6 +26,9 @@ internal class CreateSnapshotFromWebDriverIn
         [JsonProperty("captureDom")]
         public bool? CaptureDom { get; set; }
 
+        [JsonProperty("clipSelector")]
+        public string? ClipSelector { get; set; }
+
         public CreateSnapshotFromWebDriverIn(
```

Comment:
I think we could just leave the object as it is and require the user to set properties via setters. The constructor is getting bigger.

Reply:
This constructor is not used by the customer so we can modify it as we please.
I'll simplify it in another MR to rely mainly on properties.